tests/test_cluster_studio.py

```diff
@@ -0,0 +1,39 @@
+import pandas as pd
+
+from splink.cluster_studio import _get_cluster_id_by_density
+from splink.duckdb.linker import DuckDBLinker
+
+# Simple df and settings for linker
+person_ids = [i + 1 for i in range(5)]
+df = pd.DataFrame({"person_id": person_ids})
+
+settings = {
+    "link_type": "dedupe_only",
+    "unique_id_column_name": "person_id",
+}
+linker = DuckDBLinker(df, settings)
+
+# Dummy cluster metrics table
+cluster = ["A", "B", "C", "D", "E"]
+n_nodes = [3, 2, 10, 3, 19]
+n_edges = [2, 1, 5, 2, 25]
+density = [
+    (n_edges * 2) / (n_nodes * (n_nodes - 1))
+    for n_nodes, n_edges in zip(n_nodes, n_edges)
+]
+df_metrics = pd.DataFrame(
+    {"cluster_id": cluster, "n_nodes": n_nodes, "n_edges": n_edges, "density": density}
+)
+
+# Convert to Splink dataframe
+df_cluster_metrics = linker.register_table(
+    df_metrics, "df_cluster_metrics", overwrite=True
+)
+
```

Comment:
I think it might be handy to have this in the test function itself. Not a huge issue, but think it helps make the test a bit clearer exactly what is going on in the setup. Also not massive as this is quick, but any top-level code will run everytime you run testing (even if this test is not collected)

Reply:
Good to know! Thank you. Moved 👍